test/edge-cases.test.ts

```diff
@@ -71,7 +71,8 @@ describe("edge cases", () => {
     });
   });
 
-  context("try to decode invalid MessagePack binary", () => {
+  // No longer relevant because 0xc1 is used to encode `undefined`
+  context.skip("try to decode invalid MessagePack binary", () => {
```

Comment:
I guess you could enable test with another unused hexa

Reply:
Entire allowed range is used after this change, so it doesn't look possible